pkg/apis/core/types_shoot.go

```diff
@@ -636,6 +634,10 @@ type KubeAPIServerConfig struct {
 	DefaultUnreachableTolerationSeconds *int64
 	// EncryptionConfig contains customizable encryption configuration of the API server.
 	EncryptionConfig *EncryptionConfig
+	// StructuredAuthentication contains configuration settings for structured authentication to the kube-apiserver.
+	StructuredAuthentication *StructuredAuthentication
+	// StructuredAuthorization contains configuration settings for structured authorization to the kube-apiserver.
```

Comment:
```suggestion
	// StructuredAuthorization contains configuration settings for structured authorization for the kube-apiserver.
```

Nit: Sounds more natural FMPOV. Also the struct doc string says "for".